src/main/resources/bidder-config/applogy.yaml

```diff
@@ -0,0 +1,26 @@
+adapters:
+  applogy:
+    enabled: true
+    endpoint: http://localhost:8090
```

Comment:
This is not where their AD Engine is) Check `config/config.go` from GO -> adapters.applogy.endpoint

As I understand - this and `enabled=true` are to run the Integration test. The config should contain actual bidder configuration and have bidder disabled by default, 

For test, add these properties to `src/test/resources/org/prebid/server/it/test-application.properties` which are used for `SpringBootTest` environment used in integration tests.